helm/examples/azure/hpcc-azurefile/values.yaml

```diff
@@ -1,13 +1,22 @@
-# Default values for hpcc-azurefile
+# Default values for hpcc-localfile.
 
-dllStorage:
-  size: 256Mi
+planes:
+- name: dali
+  size: 1Gi
+  labels: [ "dali" ]
```

Comment:
subPath is needed (as per hpcc-localfile)

Reply:
subPath is added to schema and values.yaml